### ISSUE TITLE
Swa-Desi Addon Update Asking for Email

### ISSUE THREAD

**Issue by woodg8** [johndoe80/summer-project#3]
The Swa-desi addon update today is asking for an email now, how do i register my email to continue using the addon properly?

**Comment by heio123** [comment]
Do you want us to donate to get access?

**Comment by kanhaiyas** [comment]
Please respond about this registration thing.

**Comment by kanhaiyas** [comment]
Yes, it worked me after donating. Just send an email confirming your donation to the developer.

**Comment by woodg8** [comment]
no way am i donating, how greedy are these guys, they dont even upload the vids themselves! charging us just for linking to other videos, for shame on the devs

**Comment by heio123** [comment]
Is this addon still maintained? I donated a week ago and have no gotten a response :(

**Comment by woodg8** [comment]
you should have known not to pay these clowns, the moment they asked for donations should have been the moment it was obvious they were looking for a quick money grab before disappearing. never pay anyone like this

shame on them

**Comment by heio123** [comment]
Actually the dev was busy and it was undertsandable. My account was made the very next day after i shot them an email. Not a money grab fyi.

**Comment by kanhaiyas** [comment]
Yes, it works great. I donated about 4 months back and it works great.

**Comment by woodg8** [comment]
how do i pay, whats the email to contact the devs?
